geo.txt

```diff
@@ -56,3 +56,731 @@ Phys 91(11):957-962
 # Date first added: 2023-05-15 (by Nike Dattani)
 
 H2 Ar 3.957
+
+NAME = AgClH3N  2010
+# Point Group: C3v
+# Shape: Trigonal pyramidal --- Linear
+# Bond lengths: r0     (angstroms)
+# Bond angles:  theta0 (degrees)
+# Source Paper: Mikhailov VA, Tew DP, Walker NR, Legon AC (2010) H3N⋅⋅⋅Ag–Cl: Synthesis in a supersonic jet and
+characterization by rotational spectroscopy. Chem Phys Lett 499(1-3):16-20
+# Date first added: 2023-07-05 (by Sichao Xiong)
+
+Ag   Cl     2.26333(6)
+Ag...N      2.15444(6)
+N    H      1.0129
+H    N...Ag 113.48(2)
+
```

Comment:
If you take a look at the triatomics that I added earlier, you might notice that I didn't always report the geometry data in the same order as in the book. I reported the geometry data in such a way that the structure could be un-ambiguously inferred without needing an image. Is this possible for this molecule? Or do we absolutely need an image? Preferably the computer can just read this data and make a z-matrix, without having to refer to an image to know that NH3 is the pyramidal part and AgCl is the linear part (for example). 

These Van der Waals complexes / weakly bound systems (for which we have dotted lines) present a much more challenging case than ordinary molecules in this sense.

Reply:
Sorry for the late response. Lets do this to solve both problems (refering to the next comment) 
```
Ag   Cl     2.26333(6)
Ag...N(H3)  2.15444(6)
N    H      1.0129
H    N...Ag 113.48(2)
```
The brackets will serve to show that this wasn't originally in the book and the fact that it's the pyramidal complex that the Ag is interacting with.

Reply:
This might be confusing for some people, because parenthesis are part of the standard chemical notation for molecules, for example rhodizonic acid is: (CO)4(COH)2.

Reply:
For Ar---H2 we were given the length R_cm which is presumably the distance between the Ar atom and the centre of mass of the H2 molecule. For that reason I could just write: "H2 Ar 3.957" and it can be inferred that the length given is between the centre of mass of H2 and Ar, and I guess the fact that the point group was C2v, means that the H2---Ar complex would form a T shape rather than something less symmetric.

In the case of Ag---NH3, they have not given us the length R_cm, which would be the distance between the Ag atom and the centre of mass of the NH3 molecule. Instead they've given us the distance between N and Ag, and the N-H bond lengths. 

They have told us that the point group for the entire complex is C3v, but you were smart enough to describe the "Shape" as: Trigonal pyramidal --- Linear.

Perhaps we can treat such Van der Waals complexes by specifying each molecule in the somewhat complex separately. For example:
```
1) Trigonal pyramidal --- Linear
2) C3v --- Coov
3) N H 1.0129 --- Ag Cl 2.26333(6)
```
combined with:
```
N...Ag 2.15444(6)
```
would be enough for one to be able to reconstruct the entire molecule?

While copying and pasting the above N-H bond length, I noticed a footnote that says that the N-H bond length was "assumed" at the r_e value from CCSD(T)/cc-pVQZ, and I hope they did that CCSD(T) calculation for the whole NH3---Ag complex rather than just for NH3, but it would take longer to confirm this for certain (Section 3.3, Table 2 and the below quote are what I found so far). I'm guessing for now that the calculation included the entire complex, because otherwise they could have just used established NH3 bong lengths (and could have obtained more accurate results than CCSD(T)/cc-pVQZ). 

> "It is not possible to obtain a full r0-geometry for H3N...Ag–Cl because the distance r0(N–H) and the angle HNH are not separately determinable from the available data. We therefore fixed the value of r(N–H) at 1.0129 Å, as determined in the *ab initio* calculation described in Section 3.3 [...]"